buildout.d/versions.cfg

```diff
@@ -166,6 +166,11 @@ z3c.jbot = 0.7.2
 z3c.pt = 3.0.0a1
 z3c.recipe.usercrontab = 1.2.1
 z3c.unconfigure = 1.1
+# Todas as versões anteriores a essa dão erro no pypi, pois o buildout tenta pegar
```

Comment:
eu prefiro começar colocar esses comentários nos commits e não nas configurações e no código, pois está ficando verboso de mais por todas partes.

a descrição do PR mesmo é um lugar melhor ou um documento que descreva as mudanças na nova versão.

Reply:
Tivemos exatamente essa discussão aqui: melhor no commit ou no próprio arquivo? Achamos que explícito fica melhor porque só de bater o olho você já sabe a motivação, sem precisar de um `git blame` e ficar peneirando porque foi feito. Acaba sendo questão de preferência mesmo. Só que entre o PR e o commit, prefiro o commit, pois aí fica no histórico do repositório ao invés de um serviço como o github.

Reply:
si, tudo bem quando é um ou dois comentários, mas o negócio está ficando completamente poluído e difícil de ler.

Reply:
Ok. Iremos levar isso em consideração para as demais pinagens para ver o que for ideal.